src/organisms/cards/PlaybackCard/index.js

```diff
@@ -14,10 +15,11 @@ function PlaybackCard(props) {
     onClick,
     icon,
     editLinkText,
+    marginBottom,
   } = props;
 
   return (
-    <div className={styles['playback-card']}>
+    <div className={classnames(styles['playback-card'], marginBottom && styles['add-margin'])}>
```

Comment:
@drewdrewthis in this case I can't position from the parent because the design requires the margin only on medium-up screen sizes ([design](https://app.zeplin.io/project/58e6498f59f26ca94d348983/screen/590c9b1082a395bab0c9e57e)) - the other way I thought I might be able to solve this was to change the `Spacer` component to take a prop that would allow for the spacer to only show on certain screen sizes - what do you think?